mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -122,8 +127,11 @@ func setupWireguardDevice(devApi *device.DeviceApi, config *wireguard.ServiceCon
 	}
 
 	err = devApi.AddPeer(device.ExternalPeer{
-		PublicKey:      device.NoisePublicKey(peerPubKeyArr),
-		RemoteEndpoint: parsed,
+		PublicKey:       device.NoisePublicKey(peerPubKeyArr),
+		RemoteEndpoint:  parsed,
+		KeepAlivePeriod: 60,
+		//all traffic through this peer (unfortunatelly 0.0.0.0/0 didn't work as it was treated as ipv6)
+		AllowedIPs: []string{"0.0.0.0/1", "128.0.0.0/1"},
```

Comment:
These should be arbitrary modifiable. We should be getting those from a user somewhere.

Reply:
In the long perspective - maybe. At the moment - I don't think it's worth the effort. This "allow" to all route can be narrowed to our remote "server" only for better security maybe.